var/spack/repos/builtin/packages/zeromq/package.py

```diff
@@ -10,12 +10,13 @@ class Zeromq(AutotoolsPackage):
     """The ZMQ networking/concurrency library and core API"""
 
     homepage = "http://zguide.zeromq.org/"
-    url      = "http://download.zeromq.org/zeromq-4.1.2.tar.gz"
```

Comment:
You should still keep the URL here (maybe update to the newer URL). Spack uses this URL when you run `spack versions` and `spack checksum`.